var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -163,14 +165,25 @@ def install(self, spec, prefix):
             cmake(*cmake_args)
             make()
             make("install")
+            # install copy of host config for provenance
+            install(host_cfg_fname, prefix)
 
-    def create_host_config(self, spec, prefix):
+    def create_host_config(self, spec, prefix, py_site_pkgs_dir=None):
```

Comment:
@cyrush Sorry one last question: when is `py_site_pkgs_dir` not set by the caller? When called from `Conduit.install` it is set, and I assumed the subclass would also want to set it too.

(this question also applies to `ascent` since the same approach was taken there.

Reply:
When the python variant is off, I don't want the call to require `py_site_pkgs_dir`.

I suspect I could pass `""`, but I usually use None for optional args. 

Thinking about this, I actually need to test `~python`, I don't think it's wired up correctly in this PR

It was guarded by if `+python` in spec in the previous iteration, I lost that with the new call.

I'll pulse when that is fixed + tested.